ocaml/testsuite/tests/typing-layouts-err-msg/any.ml

```diff
@@ -0,0 +1,16 @@
+(* TEST
+   flags = "-extension layouts_alpha"
+   * expect
+*)
+
+(*********************)
+(* Any layout errors *)
+
+(* Missing_cmi *)
+(* Wildcard *)
+(* Unification_var *)
+(* Initial_typedecl_env *)
+(* Dummy_layout *)
+
+(* Type_expression_call *)
+(* Everything is a sublayout of any. Won't throw error *)
```

Comment:
I think it might be possible to make a `Missing_cmi` error, though the others probably not. But not because everything is a sublayout of `any`. If we had `t : any` and you wrote `t list`, then we'd check `any < value`, which fails, and it would print out a message.

I would say that the text associated with each of these (and any other messages we can't produce) should request folks to contact the Jane Street compilers team.